crates/driver/src/domain/competition/solution/fee.rs

```diff
@@ -31,6 +31,7 @@ use {
         competition::{
             order,
             order::{FeePolicy, Side},
+            Amounts,
```

Comment:
I think it's fine here, but in general we shouldn't be cheap with creating new structs that look similar to existing structs but different semantics (chances are at some point we might need to need a field to one of the two and it will be easier if the type isn't used across many places with different semantics).